python/lsst/cp/pipe/ptc/cpPtcSolve.py

```diff
@@ -685,30 +686,35 @@ def fitDataFullCovariance(self, dataset):
-                                                                  fitResults['fullModel']['a'],
-                                                                  fitResults['fullModel']['c'],
-                                                                  fitResults['fullModel']['noise'],
-                                                                  fitResults['fullModel']['gain'])
+            dataset.covariancesModel[ampName] = self.evalCovModel(
+                muAtAmp,
+                fitResults['fullModel']['a'],
+                fitResults['fullModel']['c'],
+                fitResults['fullModel']['noiseMatrix'],
+                fitResults['fullModel']['gain']
+            )
             dataset.covariancesSqrtWeights[ampName] = covSqrtWeightsAtAmp
             dataset.aMatrix[ampName] = fitResults['fullModel']['a']
             dataset.bMatrix[ampName] = fitResults['fullModel']['c']/fitResults['fullModel']['a']
-            dataset.covariancesModelNoB[ampName] = self.evalCovModel(muAtAmp,
-                                                                     fitResults['fullModelNoB']['a'],
-                                                                     fitResults['fullModelNoB']['c'],
-                                                                     fitResults['fullModelNoB']['noise'],
-                                                                     fitResults['fullModelNoB']['gain'],
-                                                                     setBtoZero=True)
+            dataset.covariancesModelNoB[ampName] = self.evalCovModel(
+                muAtAmp,
+                fitResults['fullModelNoB']['a'],
+                fitResults['fullModelNoB']['c'],
+                fitResults['fullModelNoB']['noiseMatrix'],
+                fitResults['fullModelNoB']['gain'],
+                setBtoZero=True
+            )
             dataset.aMatrixNoB[ampName] = fitResults['fullModelNoB']['a']
             dataset.gain[ampName] = fitResults['fullModel']['gain']
             dataset.gainErr[ampName] = fitResults['fullModel']['paramsErr'][-1]
-            readoutNoise = fitResults['fullModel']['noise'][0][0]
-            readoutNoiseSqrt = np.sqrt(np.fabs(readoutNoise))
+            readoutNoiseSquared = fitResults['fullModel']['noiseMatrix'][0][0]
+            readoutNoise = np.sqrt(np.fabs(readoutNoiseSquared))
             dataset.noise[ampName] = readoutNoise
-            readoutNoiseSigma = fitResults['fullModel']['paramsErr'][2*lenParams]
-            dataset.noiseErr[ampName] = 0.5*(readoutNoiseSigma/np.fabs(readoutNoise))*readoutNoiseSqrt
-            dataset.noiseMatrix[ampName] = fitResults['fullModel']['noise']
-            dataset.noiseMatrixNoB[ampName] = fitResults['fullModelNoB']['noise']
+            readoutNoiseSquaredSigma = fitResults['fullModel']['paramsErr'][2*lenParams]
+            noiseErr = 0.5*(readoutNoiseSquaredSigma/np.fabs(readoutNoiseSquared))*readoutNoise
```

Comment:
If I'm reading this correctly, this is the same as before just with clearer variable names.  Which means this is `noiseSquaredErr` and not `noiseErr`.